pkg/reckoner/update.go

```diff
@@ -21,17 +21,21 @@ import (
 
 func (c *Client) Update() error {
 	updatedReleases := []*course.Release{}
-	for i, r := range c.CourseFile.Releases {
-		thisReleaseDiff, err := c.diffRelease(r.Name, r.Namespace)
+	for i, release := range c.CourseFile.Releases {
```

Comment:
```suggestion
	for idx, release := range c.CourseFile.Releases {
```

For readability and then change the instances of `i` below.